qiita_db/user.py

```diff
@@ -224,10 +224,21 @@ def create(cls, email, password, info=None):
         # for sql insertion
         columns = info.keys()
         values = [info[col] for col in columns]
-
+        queue = "add_user_%s" % email
+        conn_handler.create_queue(queue)
+        # crete user
         sql = ("INSERT INTO qiita.%s (%s) VALUES (%s)" %
                (cls._table, ','.join(columns), ','.join(['%s'] * len(values))))
-        conn_handler.execute(sql, values)
+        conn_handler.add_to_queue(queue, sql, values)
+        # create user default sample holder
+        sql = ("INSERT INTO qiita.analysis "
```

Comment:
This is personal opinion but I think this highly improves code readability and even when failures happen they're easier to read on the command line. Instead of using quotes ("), I like to user triple quotes (""") and then align the SQL in a easy to read way. In this example:

```python
sql = """INSERT INTO qiita.analysis
           (email, name, description, dflt, analysis_status_id)
         VALUES (%s, %s, %s, %s, 1)"""
```
The cool thing about this, is that if the query fails, it gets also formatted on the CLI. Also it is a more natural way of reading SQL as things align better (note the small indentation on the column names).

This is not blocking though...